todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Tag(models.Model):
```

Comment:
Don't forget to describe the model fields with verbose_name so that you can understand later what kind of field it is